src/BenchmarksApps/Mvc/Startup.cs

```diff
@@ -28,6 +35,33 @@ public void ConfigureServices(IServiceCollection services)
             {
                 mvcBuilder.AddNewtonsoftJson();
             }
+
+#if JWTAUTH
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(o =>
+            {
+                o.TokenValidationParameters.ValidateActor = false;
+                o.TokenValidationParameters.ValidateLifetime = true;
+                o.TokenValidationParameters.ValidateAudience = true;
+                o.TokenValidationParameters.ValidateIssuer = true;
+                o.TokenValidationParameters.ValidIssuer = "Test";
+                o.TokenValidationParameters.ValidAudience = "test";
+                o.TokenValidationParameters.IssuerSigningKey = new SymmetricSecurityKey(Convert.FromBase64String("MFswDQYJKoZIhvcNAQEBBQADSgAwRwJAca32BtkpByiveJTwINuEerWBg2kac7sb"));
+            });
+#elif CERTAUTH
+            services.AddAuthentication(CertificateAuthenticationDefaults.AuthenticationScheme).AddCertificate(o =>
+            {
+                o.AllowedCertificateTypes = CertificateTypes.All;
+                o.RevocationFlag = X509RevocationFlag.EntireChain;
+                o.RevocationMode = X509RevocationMode.NoCheck;
+                o.ValidateCertificateUse = false;
+                o.ValidateValidityPeriod = false;
+            }).AddCertificateCache();
```

Comment:
This is the new feature you implemented in 5.0?

Reply:
Right but unless we change the load to do more than 1 request per client it will never matter, since clients aren't staying on the same connection really.  I think for the initial checkin things are fine, but we might need to look at making the load behave in more 'session' like way